app/views/users/_form_fields.html.erb

```diff
@@ -44,17 +44,17 @@
     <dd>
       <% if f.object.has_2sv? %>
         <p>2-step verification enabled</p>
-        <ul>
-          <li>
+          <p>
             <%= link_to 'Reset 2-step verification',
               reset_two_step_verification_user_path(@user),
               data: { confirm: 'Are you sure?' },
               method: :patch
               %><br/>
             Allows user to sign in without a verification code.<br/>
             User will be prompted to set up 2-step verification again the next time they sign in.
-          </li>
-        </ul>
+          </p>
```

Comment:
This `<p>` block doesn't need to be indented here (it should be the same as the indentation level on line 46).